spec/features/article_searching_spec.rb

```diff
@@ -104,7 +80,32 @@
       expect(current_url).not_to match(%r{/article\?.*&q=kittens})
     end
   end
+
+  describe 'sidenav mini-map' do
+    it 'top/bottom buttons are present in search results' do
+      article_search_for('kittens')
+
+      expect(page).to have_button('Top')
+      expect(page).to have_button('Bottom')
+    end
+  end
+
+
   it 'displays the appropriate fields in the search' do
     skip 'we need some EDS fixtures'
   end
+
+  def article_search_for(query)
+    stub_article_service(docs: StubArticleService::SAMPLE_RESULTS)
+    visit article_index_path
+
+    within '.search-form' do
+      fill_in 'q', with: query
+      if Capybara.current_driver == :poltergeist
+        find('#search').trigger('click')
+      else
+        click_button 'Search'
+      end
```

Comment:
Poltergeist tends to get tripped up by layered buttons, but Capybara doesn't seem to like `.trigger`.